### PR TITLE
Update Invite Requests README.md

### PR DESCRIPTION
**Pull request by pavlospt**

### CODE REVIEW

File: invite-to-slack/README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
 # Invite Requests
 This directory contains the script we are using in order to receive invite requests through Slack itself. This make the invitation management much much easier for admins to manage 💃
```
Comment (alexstyl) [inline invite-to-slack/README.md]: This `makes` (typo)

File: invite-to-slack/README.md [hunk 2]
```diff
@@ -1,2 +1,11 @@
...
+
+# Installation
+
+* Create a Slack invoming webhook for your Slack team: https://api.slack.com/incoming-webhooks
```
Comment (alexstyl) [inline invite-to-slack/README.md]: `incoming` (typo)

File: invite-to-slack/README.md [hunk 3]
```diff
@@ -1,2 +1,11 @@
 # Invite Requests
 This directory contains the script we are using in order to receive invite requests through Slack itself. This make the invitation management much much easier for admins to manage 💃
+
+# Installation
```
Comment (alexstyl) [inline invite-to-slack/README.md]: `##` instead. 

`#` is for the title

File: invite-to-slack/README.md (hunk 1 above)
Comment (alexstyl) [reply invite-to-slack/README.md]: Let's mention that the invites requests are done through a google form via forms.google.com.

File: invite-to-slack/README.md [hunk 4]
```diff
@@ -1,2 +1,11 @@
...
+
+* Create a Slack invoming webhook for your Slack team: https://api.slack.com/incoming-webhooks
+* Open your Invitations Google Form, go to Settings and locate `Script editor` ![Script Editor](http://image.prntscr.com/image/a69f445f4d5143449ab30b3961eb219b.png)
```
Comment (alexstyl) [inline invite-to-slack/README.md]: This is actually _not_ in the settings, but in the `More` section. 
You can phrase it like so:

Open your Invite request Google Form, and open up the `Script Editor`. In the current version of Google Forms it is located on the 'More' button like so:
screen shot 2017-04-24 at 15 47 41

File: invite-to-slack/README.md [hunk 5]
```diff
@@ -1,2 +1,11 @@
...
+* Create a new `File` and paste the content of `invite_to_slack.gs` [invite_to_slack.gs](https://github.com/androiddevs-gr/slack-tools/blob/master/invite-to-slack/invite_to_slack.gs)
```
Comment (alexstyl) [inline invite-to-slack/README.md]: Why mentioning the `invite_to_slack.gs` twice? :P

**Review by alexstyl — CHANGES_REQUESTED** [review]
Some changes to be done. Overall looks good though!

File: invite-to-slack/README.md (hunk 1 above)
Comment (pavlospt) [reply invite-to-slack/README.md]: @alexstyl I will request fix for your typos :P hehehehe

**Comment by pavlospt** [comment]
@alexstyl ping 🔔

**Review by alexstyl — APPROVED** [review]

**Comment by alexstyl** [comment]
Made a tiny change to turn the image into a hyperlink, since it looked too much.

Let's ship!

giphy-1

**Pull request merged**